### ISSUE TITLE
Do not support in the future any blocklists default, or any blocklist function the client, who needs it use external plugins like in the open internet

### ISSUE THREAD

**Issue by mx5kevin** [ZeroNetX/ZeroNet#143]
Neither similar system nor the open internet supports anything like this. This feature can be abused. Anyone who needs to solve it externally plugins. Those who want to break up this network are the main tool of this option. Smear campaign websites, the option is used for covert censorship. Anyone who needs something like this should get it externally. But the program should not centrally support anything related to censorship! Not even on the open internet, nor with competing software, there is no such thing this network has a unique centralized censor device. This is something that should not be encouraged to create block lists with anyone hundreds in one place which will eventually be full of political motivation.

**Comment by slrslr** [comment]
i disagree, the option to subscribe to a blocklist is important. The feature is not advertised by zeronet so no need to worry, only people who lookup for blocklist find there is a way to combat spam, or bad content on more effective way than visiting all bad content themself and block it one by one.

**Comment by mx5kevin** [comment]
> i disagree, the option to subscribe to a blocklist is important. The feature is not advertised by zeronet so no need to worry, only people who lookup for blocklist find there is a way to combat spam, or bad content on more effective way than visiting all bad content themself and block it one by one.

The abuse has already happened with these blocklist option and this ist just the beginning. It would be important that the base software does not support any such options like this.

There have been several abuses of these block lists. With the  first earning opportunity smashed with these blocklist slandering the website owners that they are spreading viruses. What was a Monero script what mined monero. The selection was made selectively targeting only certain content. And it didn't matter if others used it the same script. The blocklist creator she disappeared not long after. She created a fake blocklist forum in a week with a blog. And she also launched a smear campaign in ZeroTalk. This should not be supported at all. The are a centralized setup which can be filled with spam. It does a lot more harm than good. The feature is default way part by zeronet. The problem is that those who make these block lists are not well-intentioned. And it can be made in large numbers quickly, so that their makers disappear quickly. It is only suitable for selective filtering based on the sympathy of the creator of the given block list. It is a system-wide centralized censor device which is part of the basic program. Nowhere on the open internet, a similar network, or the darknet exit something like this. It does not provide any real protection. This is good if certain groups don’t like certain people then they can be silenced. Those who promote these want to integrate into the development community and work against the network.

**Comment by yanmaani** [comment]
You can't prevent people from installing plugins, now can you?

ZeroNet should not by default ship with any blocklists, but to keep the functionality in there seems reasonable. Software belongs to the users, after all, if they want to run blocklists it's their choice.
